x/staging/subspaces/types/errors.go

```diff
@@ -0,0 +1,17 @@
+package types
+
+import sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+
+// RootCodespace is the codespace for all errors defined in this package
+const RootCodespace = "subspaces"
+
+var (
+	// ErrInvalidSubspace is returned if a subspace id is not valid
+	ErrInvalidSubspace = sdkerrors.Register(RootCodespace, 1, "invalid subspace")
```

Comment:
Probably we can rename this to something like `ErrInvalidSubspaceID`?

Reply:
yeah it will be better